### PR TITLE
Unify internal observability documentation and move it to website

### PR DESCRIPTION
**Pull request by tiffany76**
Fixes https://github.com/open-telemetry/opentelemetry-collector/issues/8886.

This PR brings together three different documents from the Collector repo: [Troubleshooting](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md), [OpenTelemetry Collector Observability](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/observability.md), and [Monitoring](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md). 

**NOTE: Work on this PR is broken up into three smaller pull requests. See bullet list below for corresponding PRs.**

Per @mx-psi, this new document should describe:

- How to enable observability (#4246)
- What is our observability offering and what each metric means (#4322)
- How to effectively use this telemetry to monitor the Collector (#4529)

The merged and reworked file will be located on opentelemetry.io, with its precise location in the nav hierarchy and title to be decided. 

cc @codeboten 

---

**Preview**: https://deploy-preview-4192--opentelemetry.netlify.app/docs/collector/internal-telemetry/

### CODE REVIEW

**Comment by tiffany76** [comment]
This is still a VERY rough draft and very much a WIP. I've broken up the content into sections but done nothing else. Avert your eyes! :)

**Comment by theletterf** [comment]
Great initiative, @tiffany76 ! For a better review experience, I think it'd be great if you could split the PR into one for each file / concern.

**Comment by tiffany76** [comment]
Thanks, @theletterf! I will do that next. Traveling this week so progress may be a little slow.

**Comment by svrnm** [comment]
Great initiative! As mentioned on the related collector issue I am fine with having a troubleshooting and an observability/internal telemetry page, we can talk about the right name eventually.

File: content/en/docs/collector/internal-telemetry.md [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+      "data.http_response_code": "201"
+    }
+  }
+]
+```
+
+With the Collector running, send this payload to the Collector. For example:
+
+```console
+$ curl -X POST localhost:9411/api/v2/spans -H'Content-Type: application/json' -d @trace.json
```
Comment (chalin) [inline content/en/docs/collector/internal-telemetry.md:420]:
```suggestion
```sh
curl -X POST localhost:9411/api/v2/spans -H'Content-Type: application/json' -d @trace.json
```

File: content/en/docs/collector/internal-telemetry.md [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      address: ':8888'
+```
+
+A Grafana dashboard for these metrics can be found
+[here](https://grafana.com/grafana/dashboards/15983-opentelemetry-collector/).
```
Comment (chalin) [inline content/en/docs/collector/internal-telemetry.md:287]:
```suggestion
To visualize these metrics, you can use the [Grafana dashboard](https://grafana.com/grafana/dashboards/15983-opentelemetry-collector/), for example.
```

**Review by chalin — COMMENTED** [review]
Some drive-by comments. More copyedits to come when this is no longer in draft mode.

File: content/en/docs/collector/internal-telemetry.md [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+The `safe_rate` depends on the specific configuration being used.
+
+<!--- TODO: (Carry-over) Provide reference `safe_rate` for a few selected configurations. --->
+
+### Secondary monitoring
+
+#### Queue length
+
+Most exporters offer a
+[queue/retry mechanism](../exporter/exporterhelper/README.md) that is
```
[the remark this answers is not in the document]
Comment (tiffany76) [reply content/en/docs/collector/internal-telemetry.md]: Fixed. Thanks!

**Comment by tiffany76** [comment]
> For a better review experience, I think it'd be great if you could split the PR into one for each file / concern.

Hi @theletterf and others, I've thought about how best to break this down and have decided to leave this PR here, in draft, with the full content. I'll create new smaller PRs with each of the three sections. The first PR is up (still in draft): #4246.

Hope that works for you!

**Comment by tiffany76** [comment]
Closing this PR now that #4246, #4322, and #4529 have been merged.

**Pull request closed without merging**